### PR TITLE
[OSDOCS-4382]: adding 'standalone' wording to hosted control planes overview

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OSDOCS-4382

Link to docs preview: https://53384--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#hosted-control-planes-overview_control-plane

QE review:
- [x] QE has approved this change.
**Note:** QE is not required for this change

Additional information:

### CODE REVIEW

**Comment by zanetworker** [comment]
Lgtm

**Comment by lahinson** [comment]
/label peer-review-needed

Hi peer review squad! This is a small update to the hosted control planes overview. The main purpose of the update is to add the term "standalone" to the text and the diagram. This will match the terminology that is being used in some places in the UI for hosted control planes. QE review was not needed for this update, and it has been reviewed and approved by the PM for hosted control planes.

File: modules/hosted-control-planes-overview.adoc [hunk 1]
```diff
@@ -17,11 +17,11 @@ include::snippets/technology-preview.adoc[]
 [id="hosted-control-planes-architecture_{context}"]
 == Architecture of hosted control planes
 
-Typically, {product-title} is deployed in a coupled model, where a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane includes compute, storage, and networking where workloads and applications run.
+Typically, {product-title} is deployed in a coupled, or standalone model, where a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane includes compute, storage, and networking where workloads and applications run.
```
Comment (mburke5678) [inline modules/hosted-control-planes-overview.adoc:20]: s/or standalone/or standalone, ?

**Comment by mburke5678** [comment]
One XS potential nit. Otherwise lgtm

File: modules/hosted-control-planes-overview.adoc (hunk 1 above)
Comment (lahinson) [reply modules/hosted-control-planes-overview.adoc:20]: Good catch! There should be a comma after "standalone". I'll add that.

**Pull request merged**